mkosi/resources/mkosi.md

```diff
@@ -1384,6 +1384,42 @@ local directory:
   output to stdout will be used as the credential value. Output to stderr will be ignored.
   Credentials configured with `Credentials=` take precedence over files in `mkosi.credentials`.
 
+* The **`mkosi.repart/`** directory is used as the source for
+  systemd-repart partition definition files which are passed to
+  systemd-repart when building a disk image. If it does not exist and
+  the `RepartDirectories=` setting is not configured, mkosi will default
+  to the following partition definition files:
```

Comment:
Maybe also highlight the reverse, that if these these options are used, people will have to add configuration for these themselves.